apps/judicial-system/web/src/components/PdfButton/PdfButton.tsx

```diff
@@ -7,7 +7,8 @@ import { UserContext } from '../UserProvider/UserProvider'
 import * as styles from './PdfButton.css'
 
 interface Props {
-  caseId: string
+  caseId?: string
+  connectedCaseParentId?: string
```

Comment:
Could you please add a comment explaining this prop and why you'd use it? I think it's a bit unclear and you'd have to know the implementation details of this component to know what this prop does.

Reply:
Will do. Actually, using the term "connected case" in the client and "merged case" in the backend may cause some confusion, but I decided to follow the term already used in the client.